### PR TITLE
Fixes #886 #887 Course showing the percent of students accessing the file …

### LINKED ISSUE #886: [Resources Accessed] Course showing the percent of students accessing the file as 101
##  Expected behavior (A clear and concise description of what you expected to happen) : 
The percent of students who access the file in a course should never be above 100%.

## Describe the bug (Tell us what happens instead of the expected behavior) :
For one course, I'm seeing two files that show 101% of students in the course accessed the file.

## Steps to Reproduce :
Unclear how to reproduce yet based on the data in the course.

## Screenshots/Video (If possible, add media to help explain your problem) :
image

### PR DESCRIPTION
**Pull request by jonespm**
…as 101

- Removed initial rounding of values
- Renamed total_count to total_percent to better reflect meaning of
variable
- Improved some formatting

### CODE REVIEW

File: dashboard/views.py [hunk 1]
```diff
@@ -217,7 +217,7 @@ def resource_access_within_week(request, course_id=0):
     df['grade'] = df['current_grade'].map(gpa_map)
 
     # calculate the percentage
-    df['percent'] = round(df.groupby(['resource_id_name', 'grade'])['resource_id_name'].transform('count')/total_number_student, 2)
+    df['percent'] = df.groupby(['resource_id_name', 'grade'])['resource_id_name'].transform('count') / total_number_student
```
Comment (jonespm) [inline dashboard/views.py]: This was where the fix was, the rest was just improved formatting and cleanup.

File: assets/src/components/d3/createResourceAccessChart.js [hunk 2]
```diff
@@ -142,7 +142,7 @@ function createResourceAccessChart ({ data, width, height, domElement }) {
       .text(d => (
         ((mainYScale(d.resource_name) + mainYScale.bandwidth() / 2) < miniHeight) &&
         ((mainYScale(d.resource_name) + mainYScale.bandwidth() / 2) > 0))
-        ? d.total_count
+        ? d.total_percent
```
Comment (ssciolla) [inline assets/src/components/d3/createResourceAccessChart.js]: Were we going to add the percent sign here? (e.g. `d.total_percent  + '%'`)

Comment (jonespm) [reply assets/src/components/d3/createResourceAccessChart.js]: Yeah I had a new issue #887 filed about this one

Comment (ssciolla) [reply assets/src/components/d3/createResourceAccessChart.js]: Seems related enough to your other changes that it would fit here.

Comment (jonespm) [reply assets/src/components/d3/createResourceAccessChart.js]: I just thought it might have been controversial so didn't want to sneak it in here. ;)

Comment (jonespm) [reply assets/src/components/d3/createResourceAccessChart.js]: That's fine I can add it

**Comment by ssciolla** [comment]
@jonespm, can we merge this? I'm working on `createResourceAccessChart` and would like to rebase before I submit any PRs (I don't exactly have one ready yet though). Also, it looks like @vjcao may have tried to address the percent sign issue in a separate PR...

**Comment by jonespm** [comment]
Sorry forgot about this, it's merged.

**Pull request merged**